src/prefect/cli/deploy.py

```diff
@@ -390,6 +392,13 @@ async def _run_single_deploy(
                 )
         flow_name = flow.name
     elif flow_name:
+        app.console.print(
+            generate_deprecation_message(
+                "The `prefect deploy --flow/-f` command",
+                end_date="Jun 2023",
+                help="Use `prefect deploy` to specify an entrypoint instead.",
```

Comment:
For the deprecation message, maybe "The`--flow` option has been deprecated." or "The ability to deploy by flow name has been deprecated" 

It also looks like you're missing the new option in the help message, perhaps "Use `--entrypoint` to specify the import path of your flow instead."`

Reply:
> For the deprecation message, maybe "The--flow option has been deprecated." or "The ability to deploy by flow name has been deprecated"

Good idea! I've changed it.

> It also looks like you're missing the new option in the help message, perhaps "Use --entrypoint to specify the import path of your flow instead."`

I was thinking that too but there's no `--entrypoint` option. So `prefect deploy` is the new way.

Reply:
Ah if it's an argument we should suggest `prefect deploy <entrypoint>` instead